crates/bevy_ecs/src/system/system.rs

```diff
@@ -56,6 +57,10 @@ pub trait System: Send + Sync + 'static {
     /// Initialize the system.
     fn initialize(&mut self, _world: &mut World);
     fn check_change_tick(&mut self, change_tick: u32);
+    /// The default labels for the system
+    fn default_labels(&self) -> Vec<Box<dyn SystemLabel>> {
```

Comment:
This default reminds me that we should seal `System` soon-ish.

Reply:
I'm unconvinced that systems should be sealed, but I'll leave that discussion to the "seal systems" pr (or RFC).